CHANGES

```diff
@@ -39,8 +52,8 @@ Version 8.10
 API Changes
 ~~~~~~~~~~~
 
-- Add ``/organizations/{org}/repositories/`` endpoint.
-- Add ``/organizations/{org}/repositories/{repo}/commits/`` endpoint.
+- Add ``/organizations/{org}/repos/`` endpoint.
```

Comment:
This is odd to alter the changelog from the past. I know why you're doing it, just feels wrong.